src/main/java/edu/unc/lib/boxc/migration/cdm/services/ListProjectsService.java

```diff
@@ -139,6 +143,22 @@ private List<String> allowedActions(MigrationProject project) throws Exception {
         return allowedActions;
     }
 
+    public ObjectNode processingJobs(MigrationProject project, ObjectMapper mapper) throws Exception {
+        ObjectNode processingJobs = mapper.createObjectNode();
+
+        // velocicropter report status
+        ObjectNode velocicropterStatus = mapper.createObjectNode();
+        if (Files.exists(project.getProjectPath().resolve("processing/results/velocicroptor/job_completed"))) {
+            velocicropterStatus.put(STATUS, COMPLETED);
+            processingJobs.set("velocicropter", velocicropterStatus);
```

Comment:
should be `velocicroptor` rather than `velocicropter`. Maybe we need a constant for it somewhere since it's a mouthful, maybe a `JOB_NAME` constant in VelocicroptorRemoteJob that we should also use here https://github.com/UNC-Libraries/chompb/blob/main/src/main/java/edu/unc/lib/boxc/migration/cdm/ProcessSourceFilesCommand.java#L68